README.md

```diff
@@ -1,3 +1,24 @@
-# TODO
 
-Use this template when creating new typescript projects.
+# io-pay
+- [io-pay](#io-pay)
+  - [Prerequisites](#prerequisites)
+  - [Running](#running)
+  
+Is a [WISP](https://docs.italia.it/italia/pagopa/pagopa-specifichepagamenti-docs/it/stabile/_docs/SANP_2.2_Sez2_Cap06_ComponentiTecnicheNodo.html#componente-wisp) replacement project and it will be the first web interface to allow guest (_citizen_) payment of [pagoPa](https://www.pagopa.gov.it/) system.
+
+## Prerequisites
+
+- [yarn](https://classic.yarnpkg.com/en/docs/getting-started)
+## Running
+
+Run the following commands in your console :
+```sh
+yarn install
+yarn start
```

Comment:
@pasqualespica actually I just realized that we're missing the `start` script, can you add it please?